pkgs/by-name/pa/passh/package.nix

```diff
@@ -16,16 +16,17 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   installPhase = ''
-    mkdir -p $out/bin
-    cp ${finalAttrs.pname} $out/bin
+    runHook preInstall
+    install -Dm755 passh $out/bin/passh
+    runHook postInstall
   '';
 
-  meta = with lib; {
+  meta = {
     homepage = "https://github.com/clarkwang/passh";
     description = "Sshpass alternative for non-interactive ssh auth";
-    license = licenses.gpl3;
-    maintainers = [ maintainers.lovesegfault ];
-    mainProgram = finalAttrs.pname;
-    platforms = platforms.unix;
+    license = lib.licenses.gpl3Plus;
```

Comment:
Could you confirm that `gpl3Plus` was used, and not `gpl3` or `gpl3Only`? Sharing your investigation notes would be helpful, especially regarding license changes for existing packages.